src/test/resources/directory/file1

```diff
@@ -0,0 +1 @@
+this is file1
```

Comment:
JUnit can create temporary folders and files which get deleted after the test is run. Might be useful for testing, so that you don't need to use the real files http://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html